lib/httparty/logger/logger.rb

```diff
@@ -3,16 +3,23 @@
 
 module HTTParty
   module Logger
+    def self.formatters
+      @formatters ||= {
+        :curl => Logger::CurlLogger,
+        :apache => Logger::ApacheLogger
+      }
+    end
+
+    def self.add_formatters custom_formatters
```

Comment:
I'd probably singularize this `add_logger` or `add_formater` and only add one at a time. What should the behavior be if you provide a key that already exists? This seems to just allow overriding which could be confusing. I'm wondering if we should raise if the logger already is defined.